tensorzero-core/src/optimization/together_sft/mod.rs

```diff
@@ -375,22 +387,18 @@ impl UninitializedTogetherSFTConfig {
             weight_decay: self.weight_decay,
             suffix: self.suffix,
             // Learning rate scheduler
-            lr_scheduler_type: self.lr_scheduler_type,
-            lr_scheduler_min_lr_ratio: self.lr_scheduler_min_lr_ratio,
+            lr_scheduler: self.lr_scheduler,
             // Weights & Biases integration
             wandb_api_key: self.wandb_api_key,
```

Comment:
probably we should make more fields mandatory in TogetherSFTConfig and implement default for them